components/bl-tabs/src/index.js

```diff
@@ -47,10 +47,18 @@ export default function TabsComponent({ component, eventHandlers, pods }) {
     }
   }, [currentTabId, tabsList]);
 
+  useEffect(() => setCurrentTabId(currentTab), [currentTab]);
+
   useEffect(() => {
     podsContent.dataModel.currentTabId = currentTabId;
```

Comment:
I do not fully understand where we use `currentTabId` from `dataModel`.
I assume that @StasivPavlo is right, and it should have been used in Visibility Logic.
But taking into account that we add `Current Tab Id`  property, I assume that we could use it instead.

Reply:
Now it runs on unmount too